src/main/java/edu/cornell/kfs/module/purap/rest/resource/CuEinvoiceApiResource.java

```diff
@@ -170,22 +170,47 @@ private VendorAddress getVendorAddress(List<VendorAddress> vendorAddresses) {
     private void addPoItemsToProperties(Properties poProperties, List poItems) {
         List<Properties> poLines = new ArrayList<>();
         for (Object obj : poItems) {
-            if (obj instanceof PurApItem) {
-                PurApItem purApItem = (PurApItem)obj;
+            if (obj instanceof PurchaseOrderItem) {
+                PurchaseOrderItem poItem = (PurchaseOrderItem)obj;
                 Properties lineProps = new Properties();
-                safelyAddProperty(lineProps, CUPurapConstants.AMOUNT.toLowerCase(), purApItem.getTotalAmount().toString());
-                safelyAddProperty(lineProps, CUPurapConstants.Einvoice.QUANTITY, purApItem.getItemQuantity().toString());
-                safelyAddProperty(lineProps, CUPurapConstants.Einvoice.UNIT_OF_MEASURE, purApItem.getItemUnitOfMeasureCode());
-                safelyAddProperty(lineProps, CUPurapConstants.Einvoice.DESCRIPTION, purApItem.getItemDescription());
-                lineProps.put(CUPurapConstants.Einvoice.UNIT_PRICE, purApItem.getItemUnitPrice());
-                lineProps.put(CUPurapConstants.Einvoice.LINE_NUMBER, purApItem.getItemLineNumber());
-                lineProps.put(CUPurapConstants.Einvoice.PART_NUMBER, purApItem.getItemAuxiliaryPartIdentifier());
+                safelyAddProperty(lineProps, CUPurapConstants.AMOUNT.toLowerCase(), poItem.getTotalAmount());
```

Comment:
I would suggest adding a new CUPurapConstants.Einvoice.AMOUNT constant that has a lowercase value, so that you can use a more consistent convention for tracking the Properties keys.